sdk/src/main/java/com/qonversion/android/sdk/QUserProperties.kt

```diff
@@ -0,0 +1,10 @@
+package com.qonversion.android.sdk
+
+enum class QUserProperties(val userPropertyCode: String) {
+    Email("_q_email"),
+    Name("_q_name"),
+    KochavaDeviceId("_q_kochava_device_id"),
+    AppsFlyerUserId("_q_appsflyer_user_id"),
+    AdjustAdId("_q_adjust_adid"),
+    CustomUserId("_q_custom_user_id");
```

Comment:
Could you please implement an additional method for setting user id:

```
Qonverison. instance()?.setUserID("yourSideUserID");
```

Let's use user-property`CustomUserId` for that underhood